packages/metadata/package.json

```diff
@@ -39,7 +42,8 @@
     "README.md",
     "index.js",
     "index.d.ts",
-    "dist/src",
+    "dist*/src",
+    "dist*/index*",
     "src"
```

Comment:
Should `src` be removed ... from all files?

Reply:
No, `src` is there for source maps.